### PR TITLE
Make semi-private methods in LengthValues properly hidden

### PR DESCRIPTION
**Pull request by wilddamon**
Here I've used scoping to fully hide most methods we don't want developers to see. Some were needed in LengthValue (simpleLengthToCalcLength, addSimpleLengths, subtractSimpleLengths), so I put them in in the internal scope. I think this makes LengthValue's add and subtract methods much tidier - WDYT?

### CODE REVIEW

File: src/css-simple-length.js [hunk 1]
```diff
@@ -39,29 +39,29 @@
...
-    if (this.type != subtractedLength.type) {
+    if (length.type != subtractedLength.type) {
       throw new TypeError('CSSSimpleLength units are not the same');
     }
-    return new CSSSimpleLength((this.value - subtractedLength.value), this.type);
+    return new CSSSimpleLength(length.value - subtractedLength.value, length.type);
   };
 
-  CSSSimpleLength.prototype._asCalcLength = function() {
+  function toCalcLength(simpleLength) {
```
Comment (rjwright) [inline src/css-simple-length.js]: Can we rename this to simpleLengthToCalcLength?

File: src/css-calc-length.js [hunk 2]
```diff
@@ -104,47 +103,39 @@
 
     // Iterate through all possible length types and add their values.
     internal.objects.foreach(CSSLengthValue.LengthType, function(type) {
-      if (this[type] == null && addedLength[type] == null) {
+      if (calcLength[type] == null && addedLength[type] == null) {
```
Comment (rjwright) [inline src/css-calc-length.js]: Can this not be simplified in the same way as subtractCalcLengths?

**Comment by rjwright** [comment]
lgtm with comments

File: src/css-simple-length.js (hunk 1 above)
Comment (wilddamon) [inline src/css-simple-length.js]: Done.

File: src/css-calc-length.js (hunk 2 above)
Comment (wilddamon) [inline src/css-calc-length.js]: It can, done.

**Pull request merged**